packages/bot-skeleton/src/scratch/hooks/variable.js

```diff
@@ -0,0 +1,32 @@
+Blockly.VariableMap.prototype.createVariable = function (name, opt_type, opt_id) {
```

Comment:
@rupato-deriv let's add a comment here describing why we need to override this for future reference.